src/test/config.ts

```diff
@@ -22,13 +22,14 @@ config.helpers = {
     show: !config.TestHeadlessBrowser,
     browser: 'chromium',
     windowSize: '1300x800',
-    timeout: 70000,
-    waitForTimeout: 20000,
-    waitForAction: 2000,
-    waitForNavigation: 'networkidle0',
+    timeout: 60000,
+    waitForTimeout: 15000,
+    waitForAction: 4000,
+    waitForNavigation: 'load',
```

Comment:
Can you look into changing the option on waitForNavigation to domcontentloaded and see what happens, also try the other options on the doc:

https://playwright.dev/docs/api/class-frame#frame-wait-for-navigation